golang/cosmos/x/swingset/client/cli/tx.go

```diff
@@ -44,11 +44,16 @@ func GetTxCmd(storeKey string) *cobra.Command {
 }
 
 // GetCmdDeliver is the CLI command for sending a DeliverInbound transaction
+// containing mailbox messages.
 func GetCmdDeliver() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "deliver [json string]",
-		Short: "deliver inbound messages",
-		Args:  cobra.ExactArgs(1),
+		Use:   "deliver {<messages JSON> | @- | @<file path>}",
```

Comment:
I don't know what a "messages JSON" is, as opposed to just a "JSON". Explain what `@-` does in the long description. Each command needs to either be self-contained or have an explicit reference to conventions that can be found somewhere else.